src/lib/spreadsheets/Spreadsheet.js

```diff
@@ -20,6 +21,8 @@ export default class Spreadsheet {
   }
 
   setCells(cells) {
+    this.treeIn = new Map();
+    this.treeOut = new Map();
```

Comment:
Следует дать более однозначные имена.
```suggestion
    this.dependOn = new Map();
    this.dependenciesOf = new Map();
```